sql_ddl/db.sql

```diff
@@ -8,126 +8,114 @@ create type test_scenario_status as enum ('ACTIVE' , 'INACTIVE');
 create type test_case_result as enum ('FAILED' , 'COMPLETE');
 create type user_project_status as enum ('WATCHER' , 'DEVELOPER');
 
-create table users (
-                       id              serial primary key not null ,
-                       email           varchar(100) UNIQUE not null ,
-                       password        varchar(300) not null ,
-                       name            varchar(100) not null ,
-                       role            user_role not null ,
-                       status          user_status not null ,
-                       last_request  timestamp
+create table users
+(
+    id           serial primary key  not null,
+    email        varchar(100) UNIQUE not null,
+    password     varchar(300)        not null,
+    name         varchar(100)        not null,
+    role         user_role           not null,
+    status       user_status         not null,
+    last_request timestamp
 );
 
 create table reset_token
```

Comment:
We delete this table

Reply:
it is not my table and ddl was created by it`s owner, so I dont think I should delete it